src/method/method_invoke_java.cpp

```diff
@@ -388,35 +385,15 @@ namespace cubmethod
 	  if (stmt_type == CUBRID_STMT_SELECT)
 	    {
 	      std::uint64_t qid = current_result_info.query_id;
-	      const auto &iter = m_cursor_map.find (qid);
-	      if (iter != m_cursor_map.end ())
-		{
-		  assert (false); // should not happen
-
-		  query_cursor *cursor = iter->second;
-		  cursor->close ();
-		  xqmgr_end_query (&thread_ref, cursor->get_query_id ());
-		  delete cursor;
-		  m_cursor_map.erase (iter);
-		}
-
-	      query_cursor *cursor = nullptr;
-	      // not found, new cursor is created
 	      bool is_oid_included = current_result_info.include_oid;
-	      cursor = m_cursor_map[qid] = new (std::nothrow) query_cursor (m_group->get_thread_entry(), qid, is_oid_included);
-	      if (cursor == nullptr)
-		{
-		  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_OUT_OF_VIRTUAL_MEMORY, 1, sizeof (query_cursor));
-		  return ER_OUT_OF_VIRTUAL_MEMORY;
-		}
-
-	      cursor->open ();
+	      query_cursor *cursor = m_group->create_cursor (qid, is_oid_included);
```

Comment:
The variable cursor is defined but not referenced.

Reply:
the assertion is ommitted. I've added assertion at the following line
```
assert (cursor != nullptr);
```